telosnetwork/telos-wallet
# changing the notification mechanics after transaction

# Fixes #328

## Blocked
You should review first [#PR373](https://github.com/telosnetwork/telos-wallet/pull/373) because this PR has a rebase with that branch.

## Description
Before this change, we were giving the user a success notification too early showing a modal immediately after we send the transaction. Now we show a notification for the transaction in progress and only after confirmation do we show the big green success notification modal.

## Test scenarios
- go to [this link](https://deploy-preview-380--wallet-staging.netlify.app/)
- login on EVM
- go to send page
- prepare the form with valid data and send any amount.
  - you should first see the grey notification of sending with the form data
  - then you should receive the success notification with the explore button.

## Checklist:
